truncator-cronjob/truncate-idx.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -e
+
+# We will see if we required to truncate tables, if we found a large partition (larger than the threshold)
+# proceeed to truncate tables and recreate indices.
+
+# Get one cassandra pod so we can execute nodetool
+CASSANDRA_POD=$(oc get pods -n ${HAWKULAR_NAMESPACE} | grep hawkular-cassandra | head -n 1 |  awk '{print $1}')
+
+
+# Get largest partition of metrics_idx and metrics_tags_idx
+MAX_PARTIITON_TAGS_IDX=$(oc -n ${HAWKULAR_NAMESPACE} exec ${CASSANDRA_POD} -- nodetool tablehistograms hawkular_metrics metrics_tags_idx | grep Max | awk '{print $5}')
```

Comment:
there's a typo in these variables (PARTIITON)

Reply:
Nice catch, thanks